python/tests/pypy.sh

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
Also, we might want to remove the `# -*- coding: utf-8 -*-` lines in case Apache RAT complains about them.